src/applications/disability-benefits/2346/components/PersonalInfoBox.jsx

```diff
@@ -54,23 +54,4 @@ PersonalInfoBox.propTypes = {
   dateOfBirth: PropTypes.string.isRequired,
```

Comment:
We may want to update propTypes since they don't seem to reflect the props this component is expecting anymore 🙂